pkg/cmd/cli/cmd/set/env.go

```diff
@@ -48,6 +48,12 @@ syntax.`
   # Update all containers in all replication controllers in the project to have ENV=prod
   %[1]s env rc --all ENV=prod
 
+  # Import environment from a secret
+  %[1]s env --from-secret=mysecret dc/myapp
```

Comment:
Will a secret with binary bits be properly exposed in env?  From bash it is
not

On Tue, Jun 7, 2016 at 9:33 PM, Andy Goldstein <notifications@github.com>
wrote:

> In pkg/cmd/cli/cmd/set/env.go
> <https://github.com/openshift/origin/pull/9210#discussion_r66181131>:
>
> > @@ -48,6 +48,12 @@ syntax.`
> >    # Update all containers in all replication controllers in the project to have ENV=prod
> >    %[1]s env rc --all ENV=prod
> >
> > +  # Import environment from a secret
> > +  %[1]s env --from-secret=mysecret dc/myapp
>
> Why would you want to use the base64 encoded value from a secret?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9210/files/2dca495d9d658651af8fa61da45494ba31c65f6f#r66181131>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p-FjssWl3TaoxeCFDnMBTGLLkUtLks5qJhvZgaJpZM4IwT6g>
> .
>